hieradata/site/dev/role/rke2server.yaml

```diff
@@ -0,0 +1,3 @@
+---
+rke2::release_series: "1.31"
```

Comment:
same here 👍🏻

Reply:
Thank you for the observation makes sense to me.